spec/QueueMiddlewareSpec.php

```diff
@@ -34,7 +34,7 @@ function it_executes_a_command(Producer $producer, QueueableCommand $command)
         $this->execute($command, function() {});
     }
 
-    function it_executes_invokes_the_next_middleware(Producer $producer, Command $command, Middleware $middleware)
+    function it_executes_invokes_the_next_middleware(Producer $producer, stdClass $command, Middleware $middleware)
```

Comment:
As said, we could create a stub for that. I usually put them in a stubs directory with the name space Vendor\Package\Stub and autoloaded in dev environment. But I can do that after merging the PR.